ports/atmel-samd/boards/feather_m0_adalogger/pins.c

```diff
@@ -32,6 +32,7 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_D12), MP_ROM_PTR(&pin_PA19) },
     { MP_ROM_QSTR(MP_QSTR_D13), MP_ROM_PTR(&pin_PA17) },
     { MP_ROM_QSTR(MP_QSTR_RED_LED), MP_ROM_PTR(&pin_PA17) },
+    { MP_ROM_QSTR(MP_QSTR_VOLTAGE_MONITOR), MP_ROM_PTR(&pin_PA07) },
```

Comment:
In all of these, please move this line so it's just under the D_ name of the pin (usually D9), instead of somewhere else (e.g., see D1/TX) example.

Reply:
OK. Good idea. It's done.